pkg/metrics/metrics.go

```diff
@@ -16,6 +16,51 @@ var (
 	IAMRequestDuration *prometheus.HistogramVec
 )
 
+// InitializeMetrics initializes the metrics with a given prefix and registers them to a registry.
+func InitializeMetrics(prefix string, registry prometheus.Registerer) {
+	S3RequestsTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: prefix,
+			Name:      "s3_requests_total",
+			Help:      "Total number of S3 requests, categorized by method and status.",
+		},
+		[]string{"method", "status"},
```

Comment:
I think the S3 operation type is commonly called `action` throughout the product metrics.

Reply:
Okay, do we want only action or method and action?

Reply:
Okay, I will change that.
Thanks

Reply:
same for iam

Reply:
I will squash it, changed in https://github.com/scality/cosi-driver/pull/85/commits/0945d9e4aab99b0319c818e03d9ae9c66542108a